lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -876,10 +876,7 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
   IdentifierInfo *Name2 = Field2->getIdentifier();
   if (!::IsStructurallyEquivalent(Name1, Name2)) {
     if (Context.Complain) {
-      Context.Diag2(Owner2->getLocation(),
-                    Context.ErrorOnTagTypeMismatch
-                        ? diag::err_odr_tag_type_inconsistent
-                        : diag::warn_odr_tag_type_inconsistent)
+      Context.Diag2(Owner2->getLocation(), diag::warn_odr_tag_type_inconsistent)
```

Comment:
Could you please remove the `ErrorOnTagTypeMismatch` member from the `Context` class? It seems to be unused from now.